packages/hash/frontend/src/blocks/page/PageBlock.tsx

```diff
@@ -114,14 +124,32 @@ export const PageBlock: FunctionComponent<PageBlockProps> = ({
              * so it automatically handles focusing on closest node on margin-clicking
              */
             ".ProseMirror": {
-              padding: `0 ${PAGE_HORIZONTAL_PADDING_FORMULA} 320px`,
+              padding: `0 ${containerPadding[1]} 320px ${containerPadding[0]}`,
               minWidth: `calc(${PAGE_CONTENT_WIDTH}px + (${PAGE_MIN_PADDING}px * 2))`,
             },
             // prevents blue outline on selected nodes
             ".ProseMirror-selectednode": { outline: "none" },
           }}
         />
-        <Box id="root" ref={root} position="relative" />
+        <Box id="root" ref={root} position="relative">
+          <Box
+            sx={{
+              position: "absolute",
+              right: containerPadding[1],
+              transform: "translateX(calc(100% + 48px))",
```

Comment:
Why is this here?

Reply:
You mean why are comments being rendered here?

Reply:
No – why do you need a transform here? The styling set up on PageBlock seems very convoluted at this point. Not sure I understand what's going on.